feature-flags.json

```
@@ -1,10 +1,10 @@
 [
-	"adsPax",
-	"audienceSegmentation",
-	"conversionReporting",
-	"firstPartyMode",
-	"gm3Components",
-	"privacySandboxModule",
-	"rrmModule",
-	"rrmModuleV2"
+  "adsPax",
+  "conversionReporting",
+  "firstPartyMode",
+  "gm3Components",
+  "privacySandboxModule",
+  "rrmModule",
+  "rrmModuleV2",
+  "signInWithGoogleModule"
```

The `signInWithGoogleModule` has been removed in `develop`, we shouldn't restore it here. I'll apply this change.

```suggestion
  "rrmModuleV2"
```